rest-service/manager_rest/rest/resources_v3_1/blueprints.py

```diff
@@ -243,11 +260,12 @@ def patch(self, blueprint_id, **kwargs):
         return sm.update(blueprint)
 
 
-def _handle_blueprints_labels(sm, blueprint):
+def _handle_blueprint_labels(sm, blueprint, provided_labels):
     labels_list = get_labels_from_plan(blueprint.plan,
                                        constants.BLUEPRINT_LABELS)
-    if not labels_list:
-        return
+    if provided_labels:
```

Comment:
`provided_labels` are a list of lists in this point instead of a list of tuples as they were before they entered to the `upload_blueprint` workflow.